src/app/lib/analyticsUtils/index.js

```diff
@@ -133,3 +134,26 @@ export const getReferrer = platform => {
 
 export const sanitise = initialString =>
   initialString ? initialString.trim().replace(/\s/g, '+') : null;
+
+const isValidDateTime = dateTime => !isNaN(dateTime); // eslint-disable-line no-restricted-globals
+
+const getISODate = unixTimestamp => {
+  const date = new Date(unixTimestamp);
+
+  // if the date is before 1980, our timestamp was probably in seconds.
+  // this fixes an ares bug - ARES-758 on JIRA.
+  // if you come across this in the future, please check if it's no longer needed
+  // if so, delete this!
+  if (date.getFullYear() > 1980) {
+    return date.toISOString();
+  }
+  return new Date(unixTimestamp * 1000).toISOString();
+};
+
+export const getPublishedDatetime = (attribute, articleData) => {
```

Comment:
I believe that since this function is going to be called from `ArticleAtiParams` and `FrontPageAtiParams`, the second argument should be generic.

```suggestion
export const getPublishedDatetime = (attribute, data) => {
```

Reply:
Good catch! thanks